index.html

```diff
@@ -20,10 +20,240 @@
 
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+        </div>
 
-    -->
+        <nav class="header-banner-nav2">
+          <ul>
+            <li><a href="#">OUR TEAM</a></li>
+            <li><a href="#">GALLERY</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <hero>
+      <div class="hero-section">
+
+        <div class="hero-content">
+          <div class="hero-content-title">
+            <h1>HIGHLIGHT YOUR INNER BEAUTY</h1>
+          </div>
+          <div class="hero-content-text">
+            <p>Say goodbye to split ends and hello to healthy hair! Pamper yourself and book an appointment with Luxe
+              Hair
+              and Salon.</p>
+          </div>
+
+          <section>
+            <ul class="hero-content-box">
+              <li>
+                <section>
+                  <img src="clock.png" width="30">
+                  <h3>OPENING TIMES</h3>
+                  <p>Tue-Sat 8am-7pm Sun. 8am-5pm</p>
+                </section>
+              </li>
+              <li>
+                <section>
+                  <img src="house.png" width="30">
+                  <h3>LOCATION</h3>
+                  <p>Ottawa, ON</p>
+                </section>
+              </li>
+              <li>
+                <section>
+                  <img src="calendar.png" width="30">
+                  <h3>BOOK NOW</h3>
+                  <p>Book Online or call (613)-999-9999</p>
+                </section>
+              </li>
+            </ul>
+          </section>
+        </div>
+
+        <img class="hero-image" src="hair.png" width="960">
+      </div>
+  </div>
+  </hero>
+
+
+  <main>
+    <!--Sub Section 1: OUR PHILOSOPHY-->
+    <div class="sub-section-1">
+      <div class="section-1-title">
+        <p>OUR PHILOSOPHY</p>
+      </div>
+      <div class="section-1-content">
+        <p>At first impression, we're no different from your average salon: buzzing with blow dryers and conversations
+          driven by coffee. Yet, we like to take a more leisurely approach and slow things down. We pay attention to
+          our
+          clients, remember the names of their children just as we do with our friends, and never double-book. We
+          exclusively purchase non-toxic hair products, and 95% of our waste is recycled. In essence, we created a
+          salon
+          experience that is both opulent and environmentally friendly so that you can have your cake and eat it too.
+          <strong>Welcome to Luxe Hair and Salon</strong>
+        </p>
+      </div>
+    </div>
+    </div>
+
+    <!--Sub Section 2: OUR SERVICES-->
+    <div class="sub-section-2">
+
+      <div class="section-2-table-box">
+
+        <div class="section-2-title">
+          <p>OUR SERVICES</p>
+        </div>
+
+        <table class="section-2-table">
+          <thead>
+            <tr>
+              <td>Service</td>
+              <td>Price</td>
+            </tr>
+          </thead>
```

Comment:
So close! These should be `<th>` since they are inside the `thead`

```html
<thead>
             <tr>
               <th>Service</th>
               <th>Price</th>
             </tr>
           </thead>
```